### DISCUSSION TITLE
Have you ever changed the Dexcom G6 slot number?

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#2834] [General]
You can set xDrip to use a slot other than the default.
This is useful if you have an app that already occupies the mobile slot.  CamAPS is an example of an app that occupies the mobile slot of a G6.

You can only do this if you run xDrip on a different piece of hardware.  So, you cannot run xDrip on the same phone that runs CamAPS.

Do you use this feature?
Is it reliable?
Have you ever experienced the slot not changing as expected?
Have you ever experienced the slot changing back without you changing the setting?
Have you ever experienced an unexpected behavior?

**Comment by jbr7rr** [comment]
> Do you use this feature? 

Sometimes, I have my test phone setup to use slot 1 (or 3 when using anubis and watch)

> Is it reliable?

Somewhat, the connection on my test phone seems spotty in general. Once it has an initial connection, it seems to work. Initial connection does seem to take a bit longer. Note that I do not use this that often, so never paid too much attention to the connection. In the coming period I will pay more attention and update here

> Have you ever experienced the slot not changing as expected?

No, but I always restarted xDrip when changing slots just in case 

> Have you ever experienced the slot changing back without you changing the setting?

No

> Have you ever experienced an unexpected behavior?

No


I just tested it with the G5 I'm currently wearing, and it took about 20-25 minutes to connect. Second reading have not yet arrived after 30 min, but not sure if this is bound to the slot number, won't test with slot 2 on my test phone as I don't want to lose readings. I updated to latest nightly trough app before test

Will test again in about a week with a G6 and write an update

**Reply by jbr7rr** [reply]
Update, also with G6 I'm not able to receive the BG's simultaneously on two phones using different slot numbers. Receiving BG's when one of the phones are turned off works regardless of the slot chosen

**Comment by Emilybain** [comment]
I changed the slot number following your advice as I was having connection issues with my Dexcom One transmitter and none of the usual fixes worked. Changing the slot number did immediately allow me to connect to the transmitter and I haven't had any issues since removing the slot number. 
Thanks again for your help

**Reply by post2023** [reply]
@Emilybain 
Did I understand it correctly. 

- You have changed the Dexcom G6 slot in Xdrip+ in engineering mode from slot 2 to slot 1? 

- Did you deactivate the engineering mode again or does this have to remain active?

- Did you have to make any other settings in xdrip+ so that the connection between Xdrip+ and the Dexcom G6 is made via slot 1. 

Thank you very much. 
Greetings

**Reply by Navid200** [reply]
I advised Emily to use the feature.

You can see everything about it on the following page.
If anything on that page is not clear, or if you have any questions that is not already answered on that page, please let me know.

This is currently a feature that is only available in engineering mode because we don't feel confident about how it may behave on different devices.

https://navid200.github.io/xDrip/docs/G6_slot.html

**Comment by xJoe** [comment]
> Do you use this feature? Is it reliable? Have you ever experienced the slot not changing as expected?

I set xDrip to 1 and start the sensor. At the beginning xDrip gets readings but as soon as I use Shuggah on my iPhone in parallel xDrip looses readings.

any suggestions what I might do wrong are welcome.

**Comment by tim2000s** [comment]
I'm trying to run xDrip on a second phone using slot 1, with BYODA on primary phone - a Samsung S23 Ultra (presumably using slot 2).

I've historically run both xDrip and BYODA on the Slot 1 testing phone with no issues. I'm using the latest nightly (2nd June), but with slot 1 selected, the phone doesn't seem to be able to pair with the Transmitter (G6). 

Phone is a Redmi Note 10 Pro 5G. 

Any ideas?

**Reply by Navid200** [reply]
This could deviate from the subject of this thread.  it's always best to create a new thread when you have a question.

xDrip and BYODA both use slot 2.
Changing the slot number is not recommended.  it could lock the transmitter permanently.  That's why it is in the engineering mode.
What's in engineering mode is there because we don't think it should be used for normal use.

**Reply by Navid200** [reply]
Please open a new discussion, not a new issue.
